src/main/java/uk/gov/hmcts/reform/pcqbackend/service/ConsolidationService.java

```diff
@@ -42,9 +42,11 @@ public List<ProtectedCharacteristics> getPcqsWithoutCase(@Nullable List<String>
         log.info("Co-Relation Id : {} - getPcqsWithoutCase service invoked", coRelationId);
 
         List<ProtectedCharacteristics> returnList = protectedCharacteristicsRepository
```

Comment:
You need to ensure that number_of_days_limit is always greater than number_of_days_less_than_limit and throw exception if otherwise.

Reply:
You need a test of course.

Reply:
This condition has been added and test has been also added to prove this test works